### DISCUSSION TITLE
what should I parse in artifact when I have artifact created on runtime?

### DISCUSSION THREAD

**Discussion by TerrenceAddison** [ncipollo/release-action#127] [Q&A]
as the title says I'm having difficulty identifying what to be inputted in the artifacts field. I created an artifact using the github action [upload-artifact](https://github.com/actions/upload-artifact). And I want to add this as artifact to my release, I tried
`artifact: "artifact-name"` 
but that doesn't seem to work, what should be the correct syntax?

**Comment by ncipollo** [comment]
Hi!

The `artifacts` input expects a path to a local file, a set of paths to local files, or one or more file globs (ex - `*.png`). The files should be local to the action runner (artifacts from a previous step, etc). This action doesn't have any awareness of artifacts which have been uploaded via `upload-artifact`.

**Reply by TerrenceAddison** [reply]
Hi!
   - uses: actions/upload-artifact@v2
      with:
        name: json
        path: |
          file1
          file2
          file3
 
     - name: Upload Release
      uses: ncipollo/release-action@v1
      with:
        artifacts: "what to put here"
        token: ${{ secrets.GITHUB_TOKEN }}

          
Assuming I have this line of yml when using upload-artifact which is a step before the upload release step, how would I refer to the artifact named **json** if I want to include it to the artifacts of upload release?

**Comment by davesbrown** [comment]
Having the same issue of path artifacts: "what to put here"

I've tried explicitly putting the the full path of a txt file, as well as using $github.workspace

artifacts: |
"${{ github.workspace }}\Version.txt"

but comes up file match not found
